code/src/components/thought/Likes.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import HeartIcon from '../../assets/heart-icon.png';
+
+const Likes = ({ id, likes, onLiked }) => {
+  const LIKE_URL = `https://happy-thoughts-technigo.herokuapp.com/thougths/${id}/like`;
```

Comment:
You've got a typo in your URL and because of that your app is not posting any likes ;)
instead of `/thougths/` it should be `/thoughts/`

Reply:
How embarassing! 😰